edsc/plugins/datasource/cwic/src/js/Granules.jsx

```diff
@@ -5,6 +5,7 @@ import * as CwicUtils from './CwicUtils.jsx';
 
 let ajax = window.edsc.util.xhr.ajax;
 let clientId = window.edsc.config.cmrClientId;
+let murmurhash3 = window.edsc.util.murmurhash3;
```

Comment:
The cwic url compressor class above should be pulled into this plugin as well as the murmurhash class (unless I'm missing where it's used elsewhere)